.jenkins/actions/build_bare_metal.sh

```diff
@@ -0,0 +1,186 @@
+}
+
+# echo basic setup
+echo "####### executing: $0 $* (PID=$$ HOST=$HOSTNAME TIME=`date '+%D %H:%M:%S'`)"
+
+# start timer
+T="$(date +%s)"
+
+# parse command line options (pass all of them to function)
+parseOptions $*
+
+# check presence of env directory
+pushd `dirname $0` > /dev/null
+envloc=`/bin/pwd`/..
+popd > /dev/null
+
+# determine root directory
+pushd ${envloc}/.. > /dev/null
+rootdir=`/bin/pwd`
+popd > /dev/null
+
+# setup module environment and default queue
+# note: disable error checking since script uses fails for checks
+set +e
+. ${envloc}/env/machineEnvironment.sh
+set -e
+
+# load module tools
+. ${envloc}/env/moduleTools.sh
+
+# load git tools
+. ${envloc}/env/gitTools.sh
+
+# load scheduler tools
+. ${envloc}/env/schedulerTools.sh
+
+# compile the model
+# note: this relies on the fact that daint_gnu and daint_intel are the first
+#       two options for the configure script
+echo "### compile model"
+
+compiler_number=0
+case $compiler in
+  gnu) compiler_number=1 ;;
+  intel) compiler_number=2 ;;
+esac
+cd ${rootdir}/FV3
+echo "${compiler_number}" | ./configure
+
+module unload cudatoolkit
+module unload Boost
+module unload cray-python
+module switch PrgEnv-gnu PrgEnv-cray
+echo "==== module list ===="
+module list
+echo "====================="
+
+./compile
+
+num_exe=`/bin/ls -1d *.exe | wc -l`
+if [ "$num_exe" -lt 1 ] ; then
+   exitError 700 ${LINENO} "No valid executables have been found after compilation"
+fi
+cd -
+
+# install and run example
+# note: we setup the rundir using fv3config in a separate script in order to keep
+#       the environment of this script clean (no modules loaded etc.)
+echo "### run install and example"
+
+script=/tmp/create_rundir_$$.sh
+for config in c12_6ranks_standard c48_6ranks_standard ; do
+    cat > ${script} <<EOF
+#!/bin/bash
+set -e
+export GOOGLE_APPLICATION_CREDENTIALS=/users/olifu/.gc_cred/jenkins-sa-key.json
+cd ${rootdir}/tests/serialized_test_data_generation/
+source ${FV3CONFIG_VENV}/bin/activate
+module load gcloud
+EXPERIMENT=${config} make setup_rundir
+deactivate
+cd -
+EOF
+    chmod 755 ${script}
+    ${script}
+
+    cd ${rootdir}/tests/serialized_test_data_generation/rundir/${config}/
+
+    cp ${rootdir}/FV3/*.exe ./
+    cp ${rootdir}/FV3/conf/modules.fv3 ./module.env
+
+    sed -i 's|^ *months *= *[0-9][0-9]* *$|months = 0|g' input.nml
+    sed -i 's|^ *days *= *[0-9][0-9]* *$|days = 1|g' input.nml
+    sed -i 's|^ *hours *= *[0-9][0-9]* *$|hours = 0|g' input.nml
+    sed -i 's|^ *minutes *= *[0-9][0-9]* *$|minutes = 0|g' input.nml
+    sed -i 's|^ *seconds *= *[0-9][0-9]* *$|seconds = 0|g' input.nml
+
+    jobfile=job
+    cp ${envloc}/env/submit.${host}.${scheduler}  ./${jobfile}
+    sed -i 's|<OUTFILE>|'"${jobfile}.out"'|g' ${jobfile}
+    sed -i 's|<G2G>|'"source ./module.env; module list"'|g' ${jobfile}
+    sed -i 's|<CMD>|'"srun -n 6 ./fv3_64bit.exe"'|g' ${jobfile}
```

Comment:
where did the name of the fv3.exe change? it seems to have caused a problem running through jenkins, I think.